src/i18n/locales/en/translations.json

```diff
@@ -544,35 +544,35 @@
   "wallet.addToken.form.symbol": "Symbol",
   "wallet.addToken.form.decimals": "Amount of decimals",
   "wallet.addToken.form.assetId": "Asset ID",
-  "wallet.addToken.form.supply": "Supply on AssetHub",  
+  "wallet.addToken.form.supply": "Supply on Asset Hub",
   "wallet.addToken.form.hydrationSupply": "Supply on Hydration",
   "wallet.addToken.form.multilocation.placeholder": "Paste multillocation here",
   "wallet.addToken.form.multilocation.label": "Multilocation",
   "wallet.addToken.form.button.register.hydra": "Register on Hydration",
   "wallet.addToken.form.button.register.forMe": "Add to my UI",
-  "wallet.addToken.form.button.register.update": "Update asset data",
-  "wallet.addToken.form.button.register.acceptRisk": "I understand asset got exploited",
+  "wallet.addToken.form.button.register.update": "Update asset details",
+  "wallet.addToken.form.button.register.acceptRisk": "I understand the risk",
   "wallet.addToken.form.info": {
-    "title":"Data preview:",
+    "title":"Asset details:",
     "registered": "Asset registered on Hydration",
-    "registered.tooltip": "Asset {{value}} registered on Hydration",
-    "masterAccount": "Master account key existing",
-    "masterAccount.tooltip": "The creator of this asset has not revoked their admin rights. This means the supply can be minted, burned or frozen at any point. Proceed at your own risk.",
+    "registered.tooltip": "An indication whether the asset is already registered on Hydration.",
+    "masterAccount": "Admin rights revoked",
+    "masterAccount.tooltip": "If the asset creator has not revoked their admin rights, they can still tamper with the supply. Proceed at your own risk.",
```

Comment:
This is not right... It checks if the rights were revoked and in this case they're not so it's not if... It warns them against this